designs/titan/src/back.js

```diff
@@ -221,15 +221,80 @@ export default (part) => {
+	points.hipsOutTarget = points.waistOut.shiftTowards(points.waistIn, measurements.waistToHips).rotate(90, points.waistOut)
+	points.hipsIn = utils.beamsIntersect(points.hipsOutTarget, points.hipsInTarget, points.waistIn, points.crossSeamCurveStart)
+	points.crossSeamCurveStartMid = utils.beamsIntersect(points.crossSeamCurveStart, points.crossSeamCurveStart.shift(points.waistIn.angle(points.waistOut), 1), points.waistMid, points.seatMid)
+	if (points.seatMid.y > points.crossSeamCurveStartMid.y){
+	points.seatIn = utils.lineIntersectsCurve(points.seatMid, points.seatInTarget, points.crossSeamCurveStart, points.crossSeamCurveCp1, points.crossSeamCurveCp2, points.fork)
+	}
+	else {
+	points.seatIn = utils.beamsIntersect(points.seatMid, points.seatInTarget, points.waistIn, points.crossSeamCurveStart)
+	}
+	if (options.fitKnee) {
+	if (points.waistOut.x > points.seatOut.x) {
+	points.hipsOut = utils.lineIntersectsCurve(points.hipsOutTarget, points.hipsIn.rotate(180, points.hipsOutTarget), points.kneeOut, points.kneeOutCp2, points.seatOut, points.waistOut)
+	points.seatOutNotch = utils.lineIntersectsCurve(points.seatMid, points.seatOutTarget, points.kneeOut, points.kneeOutCp2, points.seatOut, points.waistOut)
+	}
+	else {
+	points.hipsOut = utils.lineIntersectsCurve(points.hipsOutTarget, points.hipsIn.rotate(180, points.hipsOutTarget), points.seatOut, points.seatOutCp2, points.waistOut, points.waistOut)
+	points.seatOutNotch = points.seatOut
+	}
+	points.kneeOutNotch = points.kneeOut
+	points.kneeInNotch = points.kneeIn
+	}
+	else {
+	if (points.waistOut.x > points.seatOut.x) {
+	points.hipsOut = utils.lineIntersectsCurve(points.hipsOutTarget, points.hipsIn.rotate(180, points.hipsOutTarget), points.floorOut, points.kneeOutCp2, points.seatOut, points.waistOut)
+	points.seatOutNotch = utils.lineIntersectsCurve(points.seatMid, points.seatOutTarget, points.floorOut, points.kneeOutCp2, points.seatOut, points.waistOut)
+	points.kneeOutNotch = utils.lineIntersectsCurve(points.kneeOut, points.kneeIn.rotate(180, points.kneeOut), points.floorOut, points.kneeOutCp2, points.seatOut, points.waistOut)
+	}
+	else {
+	points.hipsOut = utils.lineIntersectsCurve(points.hipsOutTarget, points.hipsIn.rotate(180, points.hipsOutTarget), points.seatOut, points.seatOutCp2, points.waistOut, points.waistOut)
+	points.seatOutNotch = points.seatOut
+	points.kneeOutNotch = utils.lineIntersectsCurve(points.kneeOut, points.kneeIn.rotate(180, points.kneeOut), points.floorOut, points.kneeOutCp2, points.seatOutCp1, points.seatOut)
+	}
+	points.kneeInNotch = utils.lineIntersectsCurve(points.kneeIn, points.kneeOut.rotate(180, points.kneeIn), points.fork, points.forkCp2, points.kneeInCp1, points.floorIn)
+	}
+	macro('sprinkle', {
+	snippet: 'notch',
+	on: ['seatOutNotch', 'kneeInNotch', 'kneeOutNotch']
+	})
+	macro('sprinkle', {
+	snippet: 'bnotch',
+	on: ['crossSeamCurveStart', 'seatIn']
+	})
```

Comment:
On the Back, you are using bnotches for these two points. However, on the Front you are using regular notches for the same points. Is this intended?

Reply:
Yes it identifies the cross seam. This seam is connected back to back whereas the others are connected to the front. One of the requests was to add notch identifiers to the back. This fufills that request.

Reply:
Okay. I still don't understand why crossSeamCurveStart and seatIn are bnotches instead of notches, and why so only on the back but not the front. However, if you say that this arrangement is intentional, then I am fine with it.